components/desktop/iso-codes/Makefile

```diff
@@ -49,6 +50,5 @@ install:	$(INSTALL_32)
 test:		$(TEST_32)
 
 # bin/xml_from_json.py script requires a Python 3 interpret
-REQUIRED_PACKAGES += runtime/python-34
```

Comment:
Shouldn't there be a new package version replacing the old, like REQUIRED_PACKAGES += runtime/python-35?

Reply:
True

Reply:
How does it find interpreter?

Reply:
It picks /usr/bin/python3